$NetBSD: patch-CMakeLists.txt,v 1.2 2022/11/14 18:44:05 adam Exp $

Don't implement sys/regset.h workaround, fix source instead.

Adapted to LLVM 23.1.2 by EmberBSD (AI-assisted); not submitted upstream.

--- CMakeLists.txt.orig
+++ CMakeLists.txt
@@ -1370,7 +1370,7 @@
 
 if( "${CMAKE_SYSTEM_NAME}" MATCHES SunOS )
    # special hack for Solaris to handle crazy system sys/regset.h
-   include_directories("${LLVM_MAIN_INCLUDE_DIR}/llvm/Support/Solaris")
+   # include_directories("${LLVM_MAIN_INCLUDE_DIR}/llvm/Support/Solaris")
 endif( "${CMAKE_SYSTEM_NAME}" MATCHES SunOS )
 
 # Make sure we don't get -rdynamic in every binary. For those that need it,
